src/main/java/lulu/command/LoadCommand.java

```diff
@@ -0,0 +1,22 @@
+package lulu.command;
+
+import lulu.TaskList;
+import lulu.UI;
+import lulu.Storage;
+
+public class LoadCommand extends Command {
+    public LoadCommand() {
```

Comment:
Since it's default for Java to use an empty constructor, should this be removed?